great_expectations/datasource/fluent/sql_datasource.py

```diff
@@ -1038,3 +1075,39 @@ def add_query_asset(
             batch_metadata=batch_metadata or {},
         )
         return self._add_asset(asset)
+
+
+def _verify_table_name_exists_and_get_normalized_typed_name_map(
```

Comment:
The map data structure is synonymous with a dictionary (see [wikipedi](https://en.wikipedia.org/wiki/Associative_array)). Could you update the name since you are returning a tuple?